test/suites/docker-compose.override.yml

```diff
@@ -0,0 +1,184 @@
+# Wikibase Suite for testing
+
+# x-common-variables: &wikibase_variables
+#   MW_ADMIN_NAME: ${MW_ADMIN_NAME}
+#   MW_ADMIN_PASS: ${MW_ADMIN_PASS}
+#   MW_ADMIN_EMAIL: ${MW_ADMIN_EMAIL}
+#   MW_WG_SECRET_KEY: ${MW_WG_SECRET_KEY}
+#   MW_WG_ENABLE_UPLOADS: ${MW_WG_ENABLE_UPLOADS}
+#   MW_WG_JOB_RUN_RATE: ${MW_WG_JOB_RUN_RATE}
+#   DB_SERVER: ${DB_SERVER}
+#   DB_USER: ${DB_USER}
+#   DB_PASS: ${DB_PASS}
+#   DB_NAME: ${DB_NAME}
+
+networks:
+  default:
+    name: wikibase-suite-test
+    external: true
+
+services:
+  # --------------------------------------------------
+  # A. CORE WIKIBASE SUITE SERVICES
+  # --------------------------------------------------
+
+  wikibase:
+    # image: ${WIKIBASE_TEST_IMAGE_URL}
+    # depends_on:
+    #   mysql:
+    #     condition: service_healthy
+    # restart: unless-stopped
+    # ports:
+    #   - ${WIKIBASE_PORT}:80
+    # volumes:
+    #   # This shares the configuration with jobrunner
+    #   - wikibase-shared:/var/www/html/:rw
+    #   - ./LocalSettings.php:/var/www/html/LocalSettings.d/LocalSettings.test.php
+    #   - quickstatements-data:/quickstatements/data
+    #   - ../${OUTPUT_DIR}:/var/log/mediawiki
+    networks:
+      default:
+        aliases:
+          - ${WIKIBASE_HOST}
```

Comment:
I think we can probably deprecate these extra network aliases and just use the service name everywhere as we have now done in Example. You probably thought of that though, and aren't doing it here yet either because this is a WIP branch or for some other good reason they should be kept :)